mechanicalsoup/stateful_browser.py

```diff
@@ -152,6 +152,24 @@ def open_relative(self, url, *args, **kwargs):
         """
         return self.open(self.absolute_url(url), *args, **kwargs)
 
+    def refresh(self):
+        """Reloads the current page with the same request as originally done.
+        Works only if page was loaded using ``StatefulBrowser`` methods.
+        The form selection and values are not retained.
```

Comment:
I first mis-read this sentence as "the values in the form obtained to get to this page are discarded", which is incorrect.

Perhaps: "Any change (``select_form``, or any value filled-in in the form) made to the current page before refresh is discarded"?